src/solver/types.jl

```diff
@@ -19,8 +19,10 @@ struct BoxModel <: AbstractModelConfig end
 struct PlaneModel <: AbstractModelConfig end
 
 abstract type AbstractSurfaceTemperature end
-Base.@kwdef struct NoSurfaceTemperature <: AbstractSurfaceTemperature end
-Base.@kwdef struct PrognosticSurfaceTemperature <: AbstractSurfaceTemperature end
+# Base.@kwdef struct NoSurfaceTemperature <: AbstractSurfaceTemperature end
+struct NoSurfaceTemperature <: AbstractSurfaceTemperature end
```

Comment:
```suggestion
struct PrescribedSurfaceTemperature <: AbstractSurfaceTemperature end
```
(I think it is more clear than NoSurfaceTemperature)